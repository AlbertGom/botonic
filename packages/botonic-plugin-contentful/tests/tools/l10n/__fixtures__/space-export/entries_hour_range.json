{
  "entries": [
    {
      "sys": {
        "space": {
          "sys": {
            "type": "Link",
            "linkType": "Space",
            "id": "5wh7etpd1y84"
          }
        },
        "id": "fAcJdXr4j6rHIokQGsnlI",
        "type": "Entry",
        "createdAt": "2019-12-10T17:49:43.659Z",
        "updatedAt": "2019-12-10T17:50:18.186Z",
        "environment": {
          "sys": {
            "id": "master",
            "type": "Link",
            "linkType": "Environment"
          }
        },
        "publishedVersion": 15,
        "publishedAt": "2019-12-10T17:50:18.186Z",
        "firstPublishedAt": "2019-12-10T17:50:18.186Z",
        "createdBy": {
          "sys": {
            "type": "Link",
            "linkType": "User",
            "id": "7pw415zrTj6gGbWJEDdbD8"
          }
        },
        "updatedBy": {
          "sys": {
            "type": "Link",
            "linkType": "User",
            "id": "7pw415zrTj6gGbWJEDdbD8"
          }
        },
        "publishedCounter": 1,
        "version": 16,
        "publishedBy": {
          "sys": {
            "type": "Link",
            "linkType": "User",
            "id": "7pw415zrTj6gGbWJEDdbD8"
          }
        },
        "contentType": {
          "sys": {
            "type": "Link",
            "linkType": "ContentType",
            "id": "hourRange"
          }
        }
      },
      "fields": {
        "name": {
          "es": "18 a 19h"
        },
        "fromHour": {
          "es": 18
        },
        "fromMinute": {
          "es": 0
        },
        "toHour": {
          "es": 19
        },
        "toMinute": {
          "es": 0
        }
      }
    }
  ],
  "locales": [
    {
      "name": "Spanish",
      "code": "es",
      "fallbackCode": null,
      "default": true,
      "contentManagementApi": true,
      "contentDeliveryApi": true,
      "optional": false,
      "sys": {
        "type": "Locale",
        "id": "4ZauuwNO1Wh7D0QbtC8tD0",
        "version": 1,
        "space": {
          "sys": {
            "type": "Link",
            "linkType": "Space",
            "id": "5wh7etpd1y84"
          }
        },
        "environment": {
          "sys": {
            "type": "Link",
            "linkType": "Environment",
            "id": "master"
          }
        },
        "createdAt": "2020-03-18T13:45:33Z",
        "updatedAt": "2020-03-18T13:45:33Z"
      }
    },
    {
      "name": "English",
      "code": "en",
      "fallbackCode": "es",
      "default": false,
      "contentManagementApi": true,
      "contentDeliveryApi": true,
      "optional": true,
      "sys": {
        "type": "Locale",
        "id": "3nYuKXPkPlkI5kTdO2BmEl",
        "version": 1,
        "space": {
          "sys": {
            "type": "Link",
            "linkType": "Space",
            "id": "5wh7etpd1y84"
          }
        },
        "environment": {
          "sys": {
            "type": "Link",
            "linkType": "Environment",
            "id": "master"
          }
        },
        "createdAt": "2020-03-18T13:45:33Z",
        "updatedAt": "2020-03-18T13:45:33Z"
      }
    }
  ]
}
