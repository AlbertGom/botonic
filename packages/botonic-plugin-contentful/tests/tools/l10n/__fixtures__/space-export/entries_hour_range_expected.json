{
  "entries": [
    {
      "sys": {
        "space": {
          "sys": {
            "type": "Link",
            "linkType": "Space",
            "id": "5wh7etpd1y84"
          }
        },
        "id": "fAcJdXr4j6rHIokQGsnlI",
        "type": "Entry",
        "createdAt": "2019-12-10T17:49:43.659Z",
        "updatedAt": "2019-12-10T17:50:18.186Z",
        "environment": {
          "sys": {
            "id": "master",
            "type": "Link",
            "linkType": "Environment"
          }
        },
        "publishedVersion": 15,
        "publishedAt": "2019-12-10T17:50:18.186Z",
        "firstPublishedAt": "2019-12-10T17:50:18.186Z",
        "createdBy": {
          "sys": {
            "type": "Link",
            "linkType": "User",
            "id": "7pw415zrTj6gGbWJEDdbD8"
          }
        },
        "updatedBy": {
          "sys": {
            "type": "Link",
            "linkType": "User",
            "id": "7pw415zrTj6gGbWJEDdbD8"
          }
        },
        "publishedCounter": 1,
        "version": 16,
        "publishedBy": {
          "sys": {
            "type": "Link",
            "linkType": "User",
            "id": "7pw415zrTj6gGbWJEDdbD8"
          }
        },
        "contentType": {
          "sys": {
            "type": "Link",
            "linkType": "ContentType",
            "id": "hourRange"
          }
        }
      },
      "fields": {
        "name": {
          "it": "18 a 19h"
        },
        "fromHour": {
          "it": 18
        },
        "fromMinute": {
          "it": 0
        },
        "toHour": {
          "it": 19
        },
        "toMinute": {
          "it": 0
        }
      }
    }
  ],
  "locales": [
    {
      "name": "it",
      "code": "it",
      "fallbackCode": null,
      "default": true,
      "contentManagementApi": true,
      "contentDeliveryApi": true,
      "optional": false,
      "sys": {
        "type": "Locale",
        "id": "3nYuKXPkPlkI5kTdO2BmEl",
        "version": 1,
        "space": {
          "sys": {
            "type": "Link",
            "linkType": "Space",
            "id": "5wh7etpd1y84"
          }
        },
        "environment": {
          "sys": {
            "type": "Link",
            "linkType": "Environment",
            "id": "master"
          }
        },
        "createdAt": "2020-03-18T13:45:33Z",
        "updatedAt": "2020-03-18T13:45:33Z"
      }
    }
  ]
}
